app/src/main/res/layout/activity_doctor_detail.xml

```diff
@@ -118,7 +145,71 @@
             android:background="@color/white"
             app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
             app:layout_constraintBottom_toBottomOf="parent"
-            app:layout_constraintTop_toBottomOf="@id/comment_title"
+            app:layout_constraintTop_toBottomOf="@+id/comment_title"
+            app:layout_constraintVertical_bias="0.0"
+            tools:layout_editor_absoluteX="0dp"
             tools:listitem="@layout/item_comment_list" />
+
+        <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
move the score to the doctor's profile block and move the comment count to the same row of comment_title